smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -91,6 +89,7 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
     } finally {
       // Check all mock expectations are satisfied.
       mockControl.$verifyAll();
+      assertTrue(chromeApiProvider.isDisposed());
```

Comment:
Also assign `chromeApiProvider` to `null`, so that the next test starts from a clear global state.